meetings/2020-01-15.md

```diff
@@ -0,0 +1,94 @@
+* Rich Trott @Trott (TSC)
+* Sam Roberts @sam-github (TSC)
+* Tobias Nießen @tniessen (TSC)
+
+## Agenda
+
+
+### Announcements
+
+* No announcements today.
+ 
+### CPC and Board Meeting Updates
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Michael: vote underway at board level for one off approval to cover travel request (see admin repo) is
+  underway to cover as we don't have the 2020 allocation finaliazed yet
+* Some discussion of travel fund at OpenJS level, if you have thoughts/concerns chimne in on
+  https://github.com/openjs-foundation/cross-project-council/pull/441 
+* Cross project Cocde of conduct team is going to be formed to discuss current proposal as MVP and then
+  what cross project extensions might make sense in terms of moderation. If you are interested please
+  join in: https://github.com/openjs-foundation/cross-project-council/issues/432
+
+
+### nodejs/node
+
+
+* process: add CLI opt to hide experimental warnings [#31000](https://github.com/nodejs/node/pull/31000)
+  * remove from agenda as TSC has proposed the direction would support
+
+* doc: add runkit embeds [#30241](https://github.com/nodejs/node/pull/30241)
+  * @tolmasky from runkit will come to next weeks meeting, we’ll defer further discussion until
+    then.
+
+
+* doc: add example for cipher.update() [#28373](https://github.com/nodejs/node/pull/28373)
+  * Sam, personally think long-form examples under every method makes them harder to read,
+    better to link to examples in other parts of the doc.
+  * Tobias, also don’t like the idea of an example in every API section.
+  * Sam from the summit some people do think an example for every API makes sense, but
+    also that they be runnable.
+  * Tobias, example is already in the doc above, so addition does not add that much more info.
+  * So TSC needs to have opinion on overall direction for the docs.
+  * Please comment in the issue, and lets let the discussion continue in GitHub.
+
+* async-hooks: introduce async-storage API [#26540](https://github.com/nodejs/node/pull/26540)
+  * Michael gave a summary of the issue
+  * Tobias, does it technically need to be in core? Some discussion around this
+  * Sam have not had time to look at issue.  But do have experience with continuous local
+    storage and believe it is something that makes sense in core as its common issue for APM
+    and need coordination with core so good to be in core.
+ * Rich we should probably move forward on getting TSC consensus. Michael has action to open
+   issue to do this.
+
+### nodejs/TSC
+
+* HackerOne monthly status meeting [#791](https://github.com/nodejs/TSC/issues/791)
+  * Sync completed during private section of meeting.
+
+### nodejs/admin
+
+* Node.js Social Media Accounts [#454](https://github.com/nodejs/admin/issues/454)
+  * defer to next week for future discussion.
+
+## Strategic Initiatives
+
+* Did not have time this week for review

```

Comment:
I would propose that we move this to something asynchronous, like the issue tracker when the meeting is open. Precious meeting time should be saved for collaborating on complicated topics. Information broadcasts should generally be done in asynchronous channels.

Reply:
I think it's been valuable to review and keep focus on those issues. Asking people to do updates offline etc. has not been successful.

Reply:
Having said that I think we could probably pare down the list to a smaller set.  Might make sense to go down to:

QUIC/HTTP3
Core Promise APIs 
Startup Performance
Build Resources